packages/rx_bloc_test/pubspec.yaml

```diff
@@ -1,22 +1,23 @@
 name: rx_bloc_test
 description: A Flutter package with the goal to enable testing rxBlocs from the FlutterRxBloc package with ease.
-version: 4.0.0
+version: 5.0.0
 repository: https://github.com/Prime-Holding/rx_bloc/tree/master/packages/rx_bloc_test
 issue_tracker: https://github.com/Prime-Holding/rx_bloc/issues
 homepage: https://primeholding.com
+topics: [bloc, state-management, testing, rxdart]
 
 environment:
   sdk: ">=3.0.5 <4.0.0"
 
 dependencies:
   fake_async: ^1.3.1
   meta: ^1.10.0
-  rx_bloc: ^5.0.0
-  rxdart: ^0.27.7
-  test: ^1.24.1
+  rx_bloc: ^6.0.0
+  rxdart: ^0.28.0
+  test: ^1.25.0
```

Comment:
Please increase only the incompatible dependencies. The test package seems to be comptabile

<img width="754" alt="image" src="https://github.com/user-attachments/assets/0449ae73-45d0-443d-a952-64cbb91fbf05">

Reply:
If you're talking about also updating rx_bloc and rxdart, would that leave the package unsupported once we release the ecosystem update?

The current package versions of `rx_bloc_test` are as follows:

```
dependencies:
  fake_async: ^1.3.1
  meta: ^1.10.0
  rx_bloc: ^6.0.0
  rxdart: ^0.28.0
  test: ^1.25.0

dev_dependencies:
  clean_coverage: ^0.0.3
  coverage: ^1.6.3
  flutter_lints: ^4.0.0
  mockito: ^5.4.2
```

Reply:
Reverted the `test` dependency.